processor/src/test/java/org/mapstruct/ap/test/verbose/CreateMapMapping.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.verbose;
+
+import java.util.Map;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+public interface CreateMapMapping {
+
+    CreateMapMapping INSTANCE = Mappers.getMapper( CreateMapMapping.class );
+
+    Map<TargetKey, TargetValue> map(Map<SourceKey, SourceValue> source);
+
+    // empty beans fail.. TODO check
```

Comment:
I saw some commits coming for this is this done now?

Reply:
I'll handle this in a separate one.. The error message is just very unclear..  Its correct that they fail, but  as a user: I wouldn't have a clue why..

Reply:
Makes sense. Maybe related to https://github.com/mapstruct/mapstruct/issues/1140?